capi/geos_c.h.in

```diff
@@ -87,19 +87,18 @@ extern "C" {
 
 
 /**
-* Type returned by GEOS_init_r(), for use in multi-threaded
-* applications.
+* Type returned by GEOS_init_r(), for use with the functions ending in `_r`
+* (the reentrant API).
 *
-* There should be only one GEOSContextHandle_t per thread.
+* Contexts must only be used from a single thread at a time.
 */
 typedef struct GEOSContextHandle_HS *GEOSContextHandle_t;
 
 /**
 * Callback function for passing GEOS error messages to parent process.
 *
-* Set the \ref GEOSMessageHandler function for error and notice messages in \ref initGEOS
-* for single-threaded programs, or using \ref initGEOS_r for threaded
-* programs
+* Set the \ref GEOSMessageHandler function for error and notice messages using
+* \ref initGEOS or \ref initGEOS_r.
```

Comment:
It seems unnecessary to me to reexplain the difference between initGEOS and initGEOS_r here, but happy to restore it in some way if folks disagree.